app/src/main/java/com/example/trainingintive/viewmodels/DogImageViewModel.kt

```diff
@@ -15,13 +18,35 @@ class DogImageViewModel @Inject constructor(
     private val _dog = MutableLiveData<String>()
     val dog: LiveData<String> = _dog
 
+    private var gettingDogImageFromApi: Disposable
+    private var localDatabaseInsertOperation: Disposable? = null
```

Comment:
Is there any reason for using two `Disposable` objects?

Reply:
I wanted to have separate variables for every operations for a case when I would want to dispose one but let continue work others